app/src/main/java/com/karumi/data/repository/NetworkSuperHeroDataSource.kt

```diff
@@ -0,0 +1,65 @@
+package com.karumi.data.repository
+
+import com.karumi.domain.model.AuthDomainError
+import com.karumi.domain.model.DomainError
+import com.karumi.domain.model.NotInternetDomainError
+import com.karumi.domain.model.SuperHero
+import com.karumi.domain.model.UnknownDomainError
+import com.karumi.marvelapiclient.CharacterApiClient
+import com.karumi.marvelapiclient.MarvelAuthApiException
+import com.karumi.marvelapiclient.model.CharacterDto
+import com.karumi.marvelapiclient.model.MarvelImage
+import com.karumi.marvelapiclient.model.MarvelResponse
+import org.funktionale.either.Either
+import java.net.ConnectException
+
+class NetworkSuperHeroDataSource(val apiClient: CharacterApiClient) : SuperHeroDataSource {
```

Comment:
The `apiClient` property should be private

Reply:
yes! sorry